src/mscorlib/src/System/IO/DriveNotFoundException.cs

```diff
@@ -18,7 +18,7 @@ namespace System.IO {
     //Thrown when trying to access a drive that is not availabe.
     [Serializable]
     [System.Runtime.InteropServices.ComVisible(true)]
-    public class DriveNotFoundException : IOException {
+    internal class DriveNotFoundException : IOException {
```

Comment:
And if is actually needed by _Error, the copy in corefx should be deleted.

The other problem is that the old `__Error` is duplicate of the new refactored `Win32Marshal.GetExceptionForWin32Error` that was copied from corefx. The mapping for `DriveNotFoundException` was removed from `Win32Marshal.GetExceptionForWin32Error` as part of this refactoring. This duplication and inconsistency should be fixed.